src/System.Net.Http/tests/StressTests/HttpStress/Dockerfile

```diff
@@ -0,0 +1,8 @@
+ARG SDK_BASE_IMAGE=mcr.microsoft.com/dotnet/core/sdk:3.0.100-buster
+FROM $SDK_BASE_IMAGE
+
+WORKDIR /app
+COPY . .
+
+ENV HTTPSTRESS_ARGS='-maxExecutionTime 30 -displayInterval 60'
```

Comment:
Why is this docker image needed at all if the only thing it does is set a env variable and then call dotnet run?

Reply:
I'm not sure I get the question. Are you asking that the app be built at image build time?

Reply:
Or are you asking why the app is containerized in the first place? The plan is to deploy the image(s) to remote machines, eventually.

Reply:
Ok, makes sense then.